pagination/templates/includes/_pagination.html

```diff
@@ -0,0 +1,15 @@
+{% load querystrings %}
+
+{% if paginator.page_range|length > 1 %}
+    <div class="pagination {{ border_class|default:"bordered" }}">
+        {% if page_obj.has_previous %}
+            <a class="prev" href="?{% querystring_update request page=page_obj.previous_page_number %}">Previous</a>
```

Comment:
I do like the explicit `?` for being a bit more obvious at a glance that you're sending some GET args.